c10/util/complex_type.h

```diff
@@ -99,18 +99,22 @@ namespace c10 {
 // - real + complex
 //
 // [Operator ==, !=]
-// 
+//
 // Each operator has three versions (taking == as example):
 // - complex == complex
 // - complex == real
 // - real == complex
-// 
+//
 // Some of them are removed on C++20, but we decide to keep them
 //
 // [Operator <<, >>]
 //
 // These are implemented by casting to std::complex
 //
+// [NOTE: Complex Operator Unification]
```

Comment:
Can we put this to the beginning of this file for better visibility?

Reply:
sure, that's a good idea.